Dockerfile

```
@@ -8,6 +8,7 @@ RUN apk add --no-cache -t build-dependencies git make gcc g++ python libtool aut
     && npm install -g node-gyp
 
 COPY ./package.json ./package-lock.json ./init-account-lookup.sql ./init-central-ledger.sql /opt/als-oracle-pathfinder/
+COPY ./src /opt/als-oracle-pathfinder/src
 
 RUN npm install --production
```

May as well update this while we're making changes in this area.
```suggestion
RUN npm ci --production
```